src/shared/utils/analytics.ts

```diff
@@ -19,25 +11,21 @@ export function initSentry() {
     return;
   }
 
-  Sentry.init({
-    dsn: SENTRY_DSN,
-    integrations: [new Integrations.BrowserTracing()],
-    tracesSampleRate: 0,
-    enabled: checkUserHasGrantedPermission(),
-    beforeSend(event) {
-      if (!checkUserHasGrantedPermission()) return null;
-      return event;
-    },
-  });
+  /* Sentry.init({ */
```

Comment:
Ok, will leave it as is to avoid increasing complexity of this PR and we can tackle it as part of https://github.com/hirosystems/stacks-wallet-web/issues/2822